backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/configuration/CassandraConfiguration.java

```diff
@@ -44,7 +46,10 @@
     public static final int DEFAULT_BLOB_PART_SIZE = 100 * 1024;
     public static final int DEFAULT_ATTACHMENT_V2_MIGRATION_READ_TIMEOUT = toIntExact(TimeUnit.HOURS.toMillis(1));
     public static final int DEFAULT_MESSAGE_ATTACHMENT_ID_MIGRATION_READ_TIMEOUT = toIntExact(TimeUnit.HOURS.toMillis(1));
-
+    public static final String DEFAULT_CONSISTENCY_LEVEL_REGULAR = "QUORUM";
```

Comment:
```suggestion
    public static final String DEFAULT_CONSISTENCY_LEVEL_QUORUM = "QUORUM";
```

Reply:
I think it is mixing the aim and the value.

Reply:
these are 3 quorum types, QUORUM is a specific QUORUM type